modules/currency.ts

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import WAWebJS from "whatsapp-web.js";
+import { send } from "../util/reply";
+
+const incorrect = "Incorrect currency code";
+const process = async (message: WAWebJS.Message, _client: WAWebJS.Client) => {
+    console.log("economy");
+    const msg = message.body.split(" ").slice(1).join(" ");
+    const arr = msg.split(" ").filter((item) => item.trim());
+    const [cur1, cur2] = arr;
+    if (!cur1) {
+        send.text(message, "Please provide currency code(s)");
+        return;
+    }
+    let res;
```

Comment:
Use ternary operator instead of if-else and make "let res" as "const res"

Reply:
Code Fixed.